client/src/utils/css-help.ts

```diff
@@ -36,9 +40,27 @@ class CSSHelp {
       ?.filter(ele => ele?.selectorText === selector)
       .map(x => x.style);
   }
-  getStyleDeclaration(selector: string): CSSStyleDeclaration | undefined {
-    return this._getStyleRules()?.find(ele => ele?.selectorText === selector)
-      ?.style;
+  getStyle(selector: string): ExtendedStyleDeclaration {
+    let len = 0;
+    const newStyle = Object.entries(
+      this._getStyleRules()?.find(ele => ele?.selectorText === selector)
+        ?.style || {}
+    )?.reduce((prev, curr) => {
+      if (Number.isInteger(Number(curr[0]))) len++;
+      // console.log(curr);
+      const _curr =
+        typeof curr[1] === 'string' ? curr[1].replace(/\s+/g, '') : curr[1];
+      return {
+        ...prev,
+        [curr[0]]: _curr
+      };
+    }, {}) as CSSStyleDeclaration;
+    return {
+      ...newStyle,
+      getPropertyValue: (prop: string): string =>
+        (newStyle?.[prop] as string) ?? '',
+      length: len
+    };
   }
```

Comment:
Unfortunately `.trim` is essentially useless to the purpose of this PR. Described in https://github.com/freeCodeCamp/freeCodeCamp/issues/42373#issuecomment-855215979 the whitespace inside the **calculated** (not custom - I am trying to use very specific wording) is meaningless, because it has already been computed, before being stripped.